code/game/objects/items/devices/traitordevices.dm

```diff
@@ -131,7 +130,7 @@ effective or pretty fucking useless.
 	popup.open()
 
 /obj/item/device/rad_laser/Topic(href, href_list)
-	if(!in_range(src, usr) || issilicon(usr) || !usr.canmove || usr.restrained())
+	if(!usr.canmove || usr.stat || usr.restrained() || !in_range(loc, usr))
```

Comment:
There's not a Topic for device, item, object, atom, or atom/movable--are you sure that would work?